src/atomicStoreFromZustand.ts

```diff
@@ -0,0 +1,32 @@
+import { createAtomicStore } from './atomicStore.js';
+import type { AtomicDefinition } from './atomicStore.js';
+
+/** Convert Zustand store definition to atomic store definition */
+export function atomicStoreFromZustand<T extends object>(
+  definition: (set: (partial: Partial<T>) => void, get: () => T) => T,
+) {
+  const state = {} as T;
+
+  const get = () => state;
+  const set = (partial: Partial<T>) => Object.assign(state, partial);
```

Comment:
This is too naive. state must be in atoms.

I don't think true "atomicStoreFromZustand" is possible. If we limit some usages, there might be a way.

Reply:
What if we added back the rootAtom to hold the basic state, and derived atoms from that?  The rootAtom would just basically be the Zustand state.

Reply:
atoms are configs and actual values are store in the Jotai store. So, there a contract mismatch. But, if we limit something, it might work. The rootAtom can be created from baseAtoms. The question is how to update the rootAtom from `set()`.